cluster/gce/upgrade.sh

```diff
@@ -447,6 +447,27 @@ if [[ "${master_upgrade}" == "false" ]] && [[ "${node_upgrade}" == "false" ]]; t
   exit 1
 fi
 
+# prompt if etcd storage media type isn't set
+if [[ -z "${STORAGE_MEDIA_TYPE:-}" ]]; then
```

Comment:
Let's not warn for etcd2, this is to prevent a specific situation. I'll make the modification and add a comment explaining.